python/ccf/ledger.py

```diff
@@ -232,21 +232,46 @@ def __next__(self):
             raise StopIteration() from e
 
 
+class Ledgerchunk:
```

Comment:
```suggestion
class LedgerChunk:
```

https://www.python.org/dev/peps/pep-0008/#type-variable-names

Reply:
Fixed it